### ISSUE TITLE
Clicking "Install metamask extension" throws an error

### ISSUE THREAD

**Issue by itzaks** [MetaMask/metamask-sdk#160]
**Describe the bug**

Using Metamask SDK a Nuxt 3 environment and using the provider with ethers.js. 

When trying to run it on a browser without metamask or any injected provider, it’s opening the onboarding to install extension. This works fine, however, when pressing "install metamask extension" it’s failing with the error: `Uncaught TypeError: Cannot read properties of undefined (reading 'debug')`

**To Reproduce**
Steps to reproduce the behaviour:
1. Open a browser without Metamask extension installed (chrome in this case)
2. Trigger the SDK using the following method:

````ts
const createMetamaskProvider = async () => {
  const sdk = new MetaMaskSDK({
    dappMetadata: {
      name: 'XXXX',
      url: 'https://XXX.XXX/icon.png'
    }
  })

  return sdk.getProvider()
}

... 
const provider = await createMetamaskProvider()
const web3Provider = new ethers.providers.Web3Provider(provider, "any")
````

4. See the inspector for error:
````js
index.js:17 Uncaught TypeError: Cannot read properties of undefined (reading 'debug')
    at startDesktopOnboarding (index.js:17:128)
    at HTMLUnknownElement.v4 (index.js:37:180250)
    at Object.xr2 (index.js:37:180575)
    at Dr2 (index.js:37:181536)
    at index.js:50:11340
    at ul2 (index.js:50:11394)
    at ll2 (index.js:50:11733)
    at cl2 (index.js:50:11816)
    at index.js:50:13710
    at index.js:50:13717
`````

**Expected behaviour**
A new window should probably open, linking to the metamask extension.

**Provide environment information**
- Platform Nuxt 3 / Vue 3.
- MetaMask SDK Version: 0.3.2
- Browser (if relevant): Experiencing this in Chrome and Safari of latest version.

**Comment by andreahaku** [comment]
Thanks for reporting. we are about to test if v0.4.0 that should be released later today fixes it as we think it does.

**Comment by christopherferreira9** [comment]
This should be fixed from v0.4.0 and v0.5.0 is already available. 
I'll be closing this one, feel free to open a new issue if you encounter issues.
Thank you for reporting.
